### ISSUE TITLE
Microsoft Office 2019 16.34 VL Crack - MinorPatch.com | MacApps Free Download

### ISSUE THREAD

**Issue by Rhinocros** [Rhinocros/minorpatch-gitalk#42]
https://www.minorpatch.com/macapps/microsoft-office.html 

 Office 2019 for Mac is designed from scratch to take advantage of the latest features of Mac, including Retina display support full – screen view and even bounce displacement.

**Comment by naveen23031984** [comment]
MS OFFICE 2019,  DOES THAT COME WITH A CRACK?

**Comment by Rhinocros** [comment]
> @naveen23031984
> MS OFFICE 2019,  DOES THAT COME WITH A CRACK?

Yes, After install Microsoft Office.pkg ,install VolumeLicense.pkg can complete crack
